### ISSUE TITLE
encrypt database passwords in application.properties file

### ISSUE THREAD

**Issue by andyxchow** [daust/JasperReportsIntegration#67]
I was able to following the instruction and bug fix article to encrypt the database passwords in the application.properties file.  However, how can I let JasperReportsIntegration know that the passwords are all encrypted.  Is there any parameter that I need to create in any configuration file to indicate that the database passwords are all encrypted in the application.properties file?

**Comment by daust** [comment]
I don't quite understand, there is nothing else to do. 

My file looks like this: 
![image](https://user-images.githubusercontent.com/4840057/101958783-4e87f200-3c04-11eb-8e07-2182e2a2f8ca.png)

The ``1:`` at the beginning indicates the encryption. The integration should decrypt it and use it. That's it. 

Do you get any error message?

**Comment by andyxchow** [comment]
It finally worked.  Somehow I had to run the encryptPasswords.cmd windows utility to encrypt the application.properties file in the server in which Tomcat is running.  I tried to run it in my own PC but those encrypted passwords didn't work.  Is it because the Java version in my PC is different than the one used by the Tomcat server?  Anyway, the encrypted passwords generated in the Tomcat server works.  Thanks.
